### PR TITLE
Fix Zeabur Doc

### PR DESCRIPTION
**Pull request by quadratz**

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]
Nice one.

**Comment by KnorpelSenf** [comment]
Why is `pnpm` used anyway?

**Comment by quadratz** [comment]
> Why is `pnpm` used anyway?

:eyes: Asking the same question

**Comment by KnorpelSenf** [comment]
> > Why is `pnpm` used anyway?
> 
> 👀 Asking the same question

Should we maybe use the opportunity and migrate it to more standard tooling (npm)?

**Comment by quadratz** [comment]
No problem

File: site/docs/hosting/zeabur-nodejs.md [hunk 1]
```diff
@@ -17,23 +17,27 @@ If you're looking for the Deno version, please check out [this tutorial](./zeabu
...
 
-To follow along, you need to have [Github](https://github.com) and [Zeabur](https://zeabur.com) accounts.
+To follow along, you need to have [GitHub](https://github.com) and [Zeabur](https://zeabur.com) accounts.
 
-### Method 1: Create a New Project From Scratch
+In this tutorial, we'll use [pnpm](https://pnpm.io) instead of npm as our package manager.
+Therefore, ensure [pnpm is installed](https://pnpm.io/installation) on your computer.
+However, if you prefer npm, you can still follow along by replacing the pnpm commands with the npm equivalents as you go.
+
+### Method 1: Create a New Project from Scratch
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:26]:
```suggestion
### Method 1: Create a New Project from Scratch
```

File: site/docs/hosting/zeabur-nodejs.md [hunk 2]
```diff
@@ -17,23 +17,27 @@ If you're looking for the Deno version, please check out [this tutorial](./zeabu
...
 
 ```sh
 # Initialize the project.
 mkdir grammy-bot
 cd grammy-bot
-pnpm init -y
+pnpm init
 
 # Install main dependencies.
 pnpm install grammy
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:37]:
```suggestion
npm install grammy
```

File: site/docs/hosting/zeabur-nodejs.md [hunk 3]
```diff
@@ -17,23 +17,27 @@ If you're looking for the Deno version, please check out [this tutorial](./zeabu
...
+### Method 1: Create a New Project from Scratch
 
 Initialize your project and install some necessary dependencies:
 
 ```sh
 # Initialize the project.
 mkdir grammy-bot
 cd grammy-bot
-pnpm init -y
+pnpm init
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:34]:
```suggestion
npm init -y
```

File: site/docs/hosting/zeabur-nodejs.md [hunk 4]
```diff
@@ -17,23 +17,27 @@ If you're looking for the Deno version, please check out [this tutorial](./zeabu
...
 
 # Install main dependencies.
 pnpm install grammy
 
 # Install development dependencies.
-pnpm install -D typescript
+pnpm install -D typescript ts-node @types/node
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:40]:
```suggestion
npm install -D typescript ts-node @types/node
```

File: site/docs/hosting/zeabur-nodejs.md [hunk 5]
```diff
@@ -75,6 +80,7 @@ Now your project's root directory should now look like this:
 │   └── bot.ts
 ├── package.json
 ├── pnpm-lock.yaml
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:82]:
```suggestion
├── package-lock.json
```

File: site/docs/hosting/zeabur-nodejs.md [hunk 6]
```diff
@@ -102,13 +110,9 @@ Our `package.json` should now be similar to this:
 Now, you can run your bot locally by running:
 
 ```sh
-pnpm start
+pnpm run start
```
Comment (quadratz) [inline site/docs/hosting/zeabur-nodejs.md:113]:
```suggestion
npm run start
```

**Review by quadratz — COMMENTED** [review]
Use npm

**Review by KnorpelSenf — APPROVED** [review]
LGTM, thank you so much

**Comment by KnorpelSenf** [comment]
We might actually be able to sync this to all languages without going through the translation process

**Review by roj1512 — APPROVED** [review]

**Comment by dzikrl** [comment]
request review, @quadratz

**Review by quadratz — COMMENTED** [review]
ID LGTM!

**Pull request merged**